cmd/minishift/cmd/start.go

```diff
@@ -324,7 +324,7 @@ func startHost(libMachineClient *libmachine.Client) *host.Host {
 	isRestart := cmdutil.VMExists(libMachineClient, constants.MachineName)
 	if !isRestart {
 		fmt.Println("-- Minishift VM will be configured with ...")
-		fmt.Println("   Memory:   ", units.HumanSize(float64((machineConfig.Memory/units.KiB)*units.GB)))
+		fmt.Println("   Memory:   ", units.BytesSize(float64(machineConfig.Memory*units.MiB)))
```

Comment:
> Right, but 5GB != 5GiB and that's what I am seeing

This is due to the suggested use of RAMInBytes, while HumanSize would use MB, This is however still not directly usable as shown in the old version:
```
fmt.Println("   Memory:   ", units.HumanSize(float64((machineConfig.Memory/units.KiB)*units.GB)))
```

> > Showing all values as MiB seems like a workaround
> For me it would be a viable option

What does this really solve?